### PR TITLE
add update function for docker agent

### PR DESCRIPTION
**Pull request by dexhunter**
* add update function
* TODO: add update.sh in upstream compose files

Signed-off-by: Dixing (Dex) Xu <dixingxu@gmail.com>

### CODE REVIEW

**Review by XuHugo — COMMENTED** [review]
ok

File: src/agent/docker-rest-agent/server.py [hunk 1]
```diff
@@ -78,6 +78,20 @@ def operate_node(id):
             elif act == 'delete':
                 container.remove()
                 res['msg'] = 'node deleted'
+            elif act == 'update':
+
+                env = {
+                'HLF_NODE_MSP': request.form.get('msp'),
```
Comment (yeasy) [inline src/agent/docker-rest-agent/server.py:84]: Need to ignore empty values.
And add test result in the commit msg.

**Review by yeasy — CHANGES_REQUESTED** [review]
The first line of commit msg should be started with capitalized letter.

**Comment by dexhunter** [comment]
@yeasy I did not find `update.sh` at [upstream scripts](https://github.com/yeasy/docker-hyperledger-fabric/tree/master/scripts), just wonder which image should I use? Thanks

**Comment by dexhunter** [comment]
The test results are ok:
image

**Comment by yeasy** [comment]
> @yeasy I did not find `update.sh` at [upstream scripts](https://github.com/yeasy/docker-hyperledger-fabric/tree/master/scripts), just wonder which image should I use? Thanks

You should create a new update.sh.

The image is built from the cello-hlf Dockerfile.

File: src/agent/docker-rest-agent/intergration-test/test.py [hunk 2]
```diff
@@ -6,18 +6,17 @@
...
 with open('msp.zip', 'rb') as node_msp, open('tls.zip', 'rb') as tls, open('block.zip', 'rb') as block, open('peer_config.zip', 'rb') as peer_config, open('orderer_config.zip', 'rb') as orderer_config:
-
     data = {
                 'msp':base64.b64encode(node_msp.read()),
                 'tls':base64.b64encode(tls.read()),
                 'bootstrap_block':base64.b64encode(block.read()),
                 'peer_config_file':base64.b64encode(peer_config.read()),
                 'orderer_config_file':base64.b64encode(orderer_config.read()),
-                'img': 'yeasy/hyperledger-fabric:2.2.0',
+                # 'img': 'yeasy/hyperledger-fabric:2.2.0',
```
Comment (dexhunter) [inline src/agent/docker-rest-agent/intergration-test/test.py:15]: @yeasy Do I need to delete this line or can I leave for a reference?

Comment (yeasy) [reply src/agent/docker-rest-agent/intergration-test/test.py:15]: Better change it to `hyperledger/cello-hlf:2.2.0`.

**Comment by dexhunter** [comment]
@yeasy ok, I've changed accordingly

**Review by yeasy — APPROVED** [review]

**Pull request merged**